spec/gen.go

```diff
@@ -3,6 +3,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:build ignore
 // +build ignore
```

Comment:
Are both of these required?

Reply:
The syntax `// +build something` is for Go 1.16 and earlier. The `//go:build something` is for 1.17+. IIRC, `go fmt` added this for me, given that we still support 1.16, according to our `go.mod` file.

[Reference](https://pkg.go.dev/cmd/go#hdr-Build_constraints):
> Go versions 1.16 and earlier used a different syntax for build constraints, with a "// +build" prefix. The gofmt command will add an equivalent //go:build constraint when encountering the older syntax.

Reply:
Thanks. So we should either remove 1.16 from `go.mod` or add it to CI.

Reply:
https://endoflife.date/go

I'll remove support in a different PR.